understanding/20/name-role-value.html

```diff
@@ -11,7 +11,7 @@ <h1>Understanding Name, Role, Value</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Authors implement components properly</dd>
+		<dt>Objective</dt><dd>People using Assistive Technology understand all components</dd>
```

Comment:
Is this needed? The AT needs to interpret the components properly so that the author's intent is conveyed to the user. However, when are people using AT ever required to even understand what a component is.

Reply:
@mgifford 
It could come down to finding a better word than 'understand', but consider this scenario: 
1. when the author assigns the correct role, the AT identifies it as, say, a menu, and the user thinks "okay, I'm in a menu" They understand what the interaction is (and some ATs will tell them basic interaction keys).
2. when the author assigns a name, the AT identifies it, and if the user is reliant on the AT presentation, she may think "Okay, this is the View menu, that's not the menu I want, so I'll proceed to the next one"
3. when the author assigns the value or state, as the user advances, she hears 'View menu, closed', moves on to the Profile menu and might hear which profile is currently selected.

That's the intended outcome of 4.1.2, so arguably, the goal is about the user getting the information. This differs a bit from 4.1.1 Parsing, where it really is just about making sure the AT is able to understand/process the code, even if there's no discernible impact on the users these days (and the reason there's no real user benefit/effect is why 4.1.1 is being 'dropped').